go/vt/wrangler/testlib/backup_test.go

```diff
@@ -196,9 +196,7 @@ func testBackupRestore(t *testing.T, cDetails *compressionDetails) error {
 		"START REPLICA",
 	}
 	sourceTablet.FakeMysqlDaemon.FetchSuperQueryMap = map[string]*sqltypes.Result{
-		"SHOW DATABASES":         {},
-		"RESET MASTER":           {},
-		"SET GLOBAL gtid_purged": {},
```

Comment:
are these two commands removed?

Reply:
@harshit-gangal No, they are not, but they moved. It's how the fake tester works, it handles things in different ways for if they are called as single statements or with a list, which is annoying here but I didn't want to untangle that issue. We still run these commands.

You can see them being added here for example:

https://github.com/vitessio/vitess/pull/15907/files#diff-4a6888c5d16d73171003445c6d3226ae39bc38d5eb43d638234b13fe03b14f1cR238-R239